packages/web5/src/web5.ts

```diff
@@ -66,9 +68,20 @@ export class Web5 {
     // create enc. keys
 
     if (!profile) {
-      // TODO: add good samaritan nodes here when they're ready
-
-      const defaultProfileDid = await this.did.create('ion');
+      const dwnUrls = await Web5.getDwnHosts();
+      let ionCreateOptions;
+
+      if (dwnUrls.length > 0) {
+        ionCreateOptions = {
```

Comment:
To enable us to use show how selecting a `keyId` for message authorization, message attestation, and record encryption work, we need the ION DIDs auto-generated to include the appropriate values as in these test fixtures:

https://github.com/TBD54566975/web5-js/blob/561f475ecf5be74e4f3cf8b530e7bc892af24860/packages/web5/tests/fixtures/test-profiles.ts#L75-L77

https://github.com/TBD54566975/web5-js/blob/561f475ecf5be74e4f3cf8b530e7bc892af24860/packages/web5/tests/fixtures/test-profiles.ts#L26-L47

Reply:
wondering if this should live [here](https://github.com/TBD54566975/web5-js/blob/main/packages/dids/src/did-ion.ts#L38-L129)

Reply:
@frankhinek i went ahead and took a stab at including these. decided not to pollute the `DidIonApi.create` method with dwn specific stuff, and instead surfaced a helper method. feel free to change anything/everything about this